Plugins/BackgroundFetch/BackgroundFetchPlugin.cs

```diff
@@ -81,16 +82,27 @@ private void RecreateObservable()
                               .ObserveOn(ThreadPoolScheduler.Instance)
                               .Subscribe(i =>
                                   {
+                                      GitArgumentBuilder args;
                                       if (_fetchAllSubmodules.ValueOrDefault(Settings))
                                       {
-                                          _currentGitUiCommands.GitModule.RunGitCmd("submodule foreach --recursive git fetch --all");
+                                          args = new GitArgumentBuilder("submodule")
+                                        {
+                                            "foreach",
+                                            "--recursive",
+                                            "git",
+                                            "fetch",
+                                            "--all"
+                                        };
```

Comment:
mis-indented

Reply:
![image](https://user-images.githubusercontent.com/3629489/46256812-e6fb8700-c47d-11e8-8383-aa2c98a2eefe.png)

Reply:
The screenshot shows a different piece of code.
The code above looks untidy and inconsistent

Reply:
Each red mark is one indention  
![image](https://user-images.githubusercontent.com/3629489/46926474-eaa31800-cffe-11e8-90dd-60e488f3f014.png)